### ISSUE TITLE
@template doesn't actually show in docs

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#23696]
I don't know if this is intentional or not, but this:

```
  /// {@template flutter.widgets.widgetsApp.title}                                                                             
  /// A one-line description used by the device to identify the app for the user.                                              
  ///                                                                                                                          
  /// On Android the titles appear above the task manager's app snapshots which are                                            
  /// displayed when the user presses the "recent apps" button. On iOS this                                                    
  /// value cannot be used. `CFBundleDisplayName` from the app's `Info.plist` is                                               
  /// referred to instead whenever present, `CFBundleName` otherwise.                                                          
  ///                                                                                                                          
  /// To provide a localized title instead, use [onGenerateTitle].                                                             
  /// {@endtemplate}                                                                                                           
  final String title;                                                                                                          
```

...results in no docs:

https://master-docs-flutter-io.firebaseapp.com/flutter/widgets/WidgetsApp/title.html

cc @gspencergoog @jcollins-g

**Comment by jcollins-g** [comment]
https://github.com/dart-lang/dartdoc/issues/1812

**Comment by zoechi** [comment]
@jcollins-g can this be closed. Looks like you fixed dart-lang/dartdoc#1812 a while ago.

**Comment by gspencergoog** [comment]
Yes, this is fixed.
